stable/redis/README.md

```diff
@@ -43,39 +49,89 @@ The command removes all the Kubernetes components associated with the chart and
-| `resources`                   | CPU/Memory resource requests/limits               | Memory: `256Mi`, CPU: `100m`                            |
-| `metrics.enabled`             | Start a side-car prometheus exporter              | `false`                                                 |
-| `metrics.image`               | Exporter image                                    | `oliver006/redis_exporter`                              |
-| `metrics.imageTag`            | Exporter image                                    | `v0.11`                                                 |
-| `metrics.imagePullPolicy`     | Exporter image pull policy                        | `IfNotPresent`                                          |
-| `metrics.resources`           | Exporter resource requests/limit                  | Memory: `256Mi`, CPU: `100m`                            |
-| `nodeSelector`                | Node labels for pod assignment                    | {}                                                      |
-| `tolerations`                 | Toleration labels for pod assignment              | []                                                      |
-| `networkPolicy.enabled`       | Enable NetworkPolicy                              | `false`                                                 |
-| `networkPolicy.allowExternal` | Don't require client label for connections        | `true`                                                  |
-| `service.annotations`         | annotations for redis service                     | {}                                                      |
-| `service.loadBalancerIP`      | loadBalancerIP if service type is `LoadBalancer`  | ``                                                      |
-| `securityContext.enabled`     | Enable security context                           | `true`                                                  |
+| Parameter                                  | Description                                                                                                    | Default                              |               |
+|--------------------------------------------|----------------------------------------------------------------------------------------------------------------|--------------------------------------|---------------|
+| `image.registry`            | Rabbitmq Image registry                                 | `docker.io`                                              |
+| `image.repository`          | Rabbitmq Image name                                     | `bitnami/rabbitmq`                                       |
+| `image.tag`                 | Rabbitmq Image tag                                      | `{VERSION}`                                              |
+| `image.pullPolicy`          | Image pull policy                                       | `Always` if `imageTag` is `latest`, else `IfNotPresent`  |
```

Comment:
Default value column should say `Always`, the '`Always` if `imageTag` is `latest`, else `IfNotPresent` ' is a thing of the past